src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -27,17 +26,56 @@ const ActorInit Bg_Inibs_Movebg_InitVars = {
     (ActorFunc)BgInibsMovebg_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96578[] = {
+extern Gfx D_060062D8[];
+extern Gfx D_06001DC0[];
+extern Gfx D_06006140[];
+extern Gfx D_06001C10[];
+extern AnimatedMaterial D_06006858;
+extern AnimatedMaterial D_06002598;
```

Comment:
If these are all from the same file, can you sort them by address, please? It'll make it easier for the person doing the object.